app/app_main.c

```diff
@@ -429,16 +429,102 @@ int main(int argc, char **argv)
     /*
      * Enable HMAC
      */
+     char value[] = "test_prereq_val";
```

Comment:
Since the valid values are "same" or a numerical value, you may want to just put "same" here so the server will accept it.